test/v1alpha1/configuration.go

```diff
@@ -63,18 +64,24 @@ func WaitForConfigLatestRevision(clients *test.Clients, names test.ResourceNames
 	err := WaitForConfigurationState(clients.ServingAlphaClient, names.Config, func(c *v1alpha1.Configuration) (bool, error) {
 		if c.Status.LatestCreatedRevisionName != names.Revision {
 			revisionName = c.Status.LatestCreatedRevisionName
+			// We also check that the revision is pinned, meaning it's not a stale revision.
+			// Without this it might happen that the latest created revision is later overriden by a newer one
```

Comment:
<!--typo-bot-->
Found potential misspelling(s):
```suggestion
			// Without this it might happen that the latest created revision is later overridden by a newer one
```